ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/EvaluatorImpl.java

```diff
@@ -131,6 +131,29 @@ public Optional<ShellReturnValue> getValue(Optional<PackageCompilation> compilat
         }
     }
 
+    @Override
+    public Optional<Object> getValueAsObject(Optional<PackageCompilation> compilation) throws
+            BallerinaShellException {
+        try {
+            return invoker.execute(compilation);
+        } catch (InvokerPanicException e) {
+            addAllDiagnostics(invoker.diagnostics());
+            invoker.resetDiagnostics();
+            throw e;
+        } catch (InvokerException e) {
+            addAllDiagnostics(invoker.diagnostics());
+            invoker.resetDiagnostics();
+            return Optional.of("");
```

Comment:
Why do we need to send an empty string Optional for a InvokerException?

Reply:
This was done to avoid having duplicate catch blocks. Since we don't require it here, removed a catch block in `76a07d1`